site/docs/v1/tech/tutorials/integrate-python-flask.md

```diff
@@ -0,0 +1,279 @@
+
+First, create the required files:
+
+```shell
+touch requirements.txt setup.py
+```
+
+Now, cut and paste the following requirements into `requirements.txt`:
+
+```text
+{% remote_include https://raw.githubusercontent.com/ritza-co/fusionauth-flask-integration/main/requirements.txt %}
+```
+
+Then, copy and paste the following code into the `setup.py` file.
+
+```python
+{% remote_include https://raw.githubusercontent.com/ritza-co/fusionauth-flask-integration/main/setup.py %}
+```
+
+Then, you can run the setup script. Please note that this setup script is designed to run on a newly installed FusionAuth instance with only one user and no tenants other than "Default". To follow this guide on a FusionAuth instance that does not meet these criteria, you may need to modify the above script. Refer to the [Python client library](/docs/v1/tech/client-libraries/python) documentation for more information.
+
+You’ll use a virtual environment `venv` to keep your workspace clean.
+
+```shell
+python -m venv venv && \
+source venv/bin/activate && \
+pip install -r requirements.txt
+```
+
+Now run the setup script, replacing `<your API key>` with the value of the API key noted earlier.
+
+```shell
+fusionauth_api_key=<your API key> python setup.py
+```
+
+If you are using PowerShell, you will need to set the environment variable in a separate command before executing the script.
+
+```shell
+$env:fusionauth_api_key_1='<your API key>'
+python setup.py
+```
+
+This configures FusionAuth for your {{page.technology}} application.
+
+When you’re done, you can log in to your instance and examine the new application configuration, the script created for you.
+
+## Create Your {{page.technology}} Application
+
+Now you are going to create a {{page.technology}} application. While this section builds a simple {{page.technology}} application, you can use the same configuration to integrate your complex {{page.technology}} application with FusionAuth.
+
+First, make a directory:
+
+```shell
+mkdir setup-flask && cd setup-flask
+```
+
+Now, create a new `requirements.txt` file to include {{page.technology}}. Add the `flask`, `python-dotenv`, and `authlib` plugins so that your requirements file looks like this:
+
+```text
+{% remote_include https://raw.githubusercontent.com/ritza-co/fusionauth-flask-integration/main/setup-flask/requirements.txt %}
+```
+
+Then, rerun `pip install` to install these new packages.
+
+```shell
+pip install -r requirements.txt
+```
+
+Next, create a file called `.env` and insert the following into it.
+
+```ini
+CLIENT_ID=e9fdb985-9173-4e01-9d73-ac2d60d1dc8e
+CLIENT_SECRET=change-this-in-production-to-be-a-real-secret
+ISSUER=http://localhost:9011
+APP_SECRET_KEY=my_super_secret_key_that_needs_to_be_changed
+```
+
+You can now start writing the code for your Flask application. Create a new file called `server.py` and add the following import statements into it:
+
+```python
+import json
```

Comment:
For these, we tend to put the whole file in at once, not breaking it down addition by addition.

This means you can use remote_include to pull in the entire file and you don't need to breakdown what is happening.